oneanddone/users/views.py

```diff
@@ -151,6 +151,16 @@ def login_failure(self, *args, **kwargs):
             logging in, let us know by emailing <a href="mailto:{email}">{email}</a>.
         """).format(email='oneanddone@mozilla.com'), extra_tags='safe')
         return super(Verify, self).login_failure(*args, **kwargs)
+    
+    def login_success(self, *args, **kwargs):
+        if (self.request.user.is_authenticated() and
```

Comment:
One of the reasons this is not working is that `self.request.user.is_authenticated()` will always return `False` here because the user has not been authenticated yet. You are calling this code before the parent `Verify` has had a chance to authenticate the user. Therefore, you need to remove this first part of the `if` statement from here and also from line 159. 

Because this is `login_success` we can assume that the user is about to be authenticated, so we don't need to check for that at all.